### DISCUSSION TITLE
Possibile recuperare anche lo stato delle batterie?

### DISCUSSION THREAD

**Discussion by Simone000** [aturri/ha-zcsazzurro#18] [Q&A] [answered]
per ora vedo l'inverter ma non le batterie, ho solo 4 dati, sostanzialmente sulla produzione.

Sarebbe possibile recuperare anche lo stato delle batterie e magari anche l'energia immessa in rete?

**Comment by aturri** [comment, marked as answer]
Buongiorno,
si, andando sulla pagina del dispositivo Impostazioni > Integrazioni > ZCS Azzurro e selezionando l'inverter, sul tab Sensori ci sono anche quelli relativi alle batterie ed energia esportata. Si trovano sotto i sensori nascosti e vanno abilitati manualmente. Homeassistant potrebbe impiegare anche 30s a rendere i dati disponibili.

<img width="1216" alt="Screenshot 2023-09-23 at 10 00 53" src="https://github.com/aturri/ha-zcsazzurro/assets/6987093/474a1352-5e19-4a5e-8959-c72f8b58217c">

**Comment by Simone000** [comment]
grazie! funziona, unica cosa non capisco il Batteria 2, in quale caso è presente?
io ho 2 batterie ma come da sito le vede come una sola

**Comment by tommasinigiovanni** [comment]
Buongiorno a tutti,
Io ho 8 entità attive su 20 e non riesco ad attivare le altre perché non vedo il tasto "ABILITA"
<img width="590" alt="image" src="https://github.com/user-attachments/assets/28458879-21b2-4477-a958-eb6df5b4b92f" />
ho provato anche a riavviare homeassistant.

Ho provato a scaricare la diagnostica e li i dati mi sembra ci siano tutti
![image](https://github.com/user-attachments/assets/fbae5385-ffea-4845-9434-73b26b44d923)

grazie mille
